e2e/cypress/integration/archived_channel/archive_channel_post_spec.js

```diff
@@ -47,4 +56,62 @@ describe('Archived channels', () => {
         // * RHS text box should not be visible
         cy.get('#reply_textbox').should('not.be.visible');
     });
+
+    it('MM-T1722 Can click reply arrow on a post from archived channel, from saved posts list', () => {
+        // # Create a channel that will be archived
+        cy.apiCreateChannel(testTeam.id, 'archived-channel', 'Archived Channel').then(({channel}) => {
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+
+            // # Post message
+            cy.postMessage('Test');
+
+            // # Create permalink to post
+            cy.getLastPostId().then((id) => {
+                const permalink = `${Cypress.config('baseUrl')}/${testTeam.name}/pl/${id}`;
+
+                // # Click on ... button of last post
+                cy.clickPostDotMenu(id);
+
+                // # Click on "Copy Link"
+                cy.uiClickCopyLink(permalink);
```

Comment:
Do we even need this step, along with clicking on dot menu?.

Reply:
yes! Otherwise we can't find the permalinked post